fog/report/types/src/lib.rs

```diff
@@ -9,22 +9,47 @@
 
 extern crate alloc;
 
+use ::prost::Message;
 use alloc::{collections::BTreeMap, string::String, vec::Vec};
-use mc_attest_core::VerificationReport;
+use mc_attest_verifier_types::{prost, VerificationReport};
 use mc_crypto_digestible::Digestible;
-use prost::Message;
 use serde::{Deserialize, Serialize};
 
+/// The attestation evidence variants for a report.
+#[derive(Clone, ::prost::Oneof, Deserialize, Eq, PartialEq, Serialize, Digestible)]
+#[digestible(transparent)]
+pub enum AttestationEvidence {
+    /// The attestation evidence is a [VerificationReport].
+    #[prost(message, tag = 2)]
+    VerificationReport(VerificationReport),
+    /// DCAP evidence
+    #[prost(message, tag = 4)]
+    DcapEvidence(prost::DcapEvidence),
+}
```

Comment:
Because of the disparate tag numbers we can't use the same as `BlockMetadataContents`